### DISCUSSION TITLE
Can we do hash class name more shorter?

### DISCUSSION THREAD

**Discussion by IRediTOTO** [chakra-ui/panda#2034] [Q&A]
Hi, Can we have hash classname like a,b,c,d? As I see right now, Panda generates a classname hash of about 6 characters. Having a class name of type a,b,c,aa,bb, will reduce the number of classnames in the HTML by about 2/3. That's significant, right?

**Comment by segunadebayo** [comment]
We're open to a better hashing algorithm by the community as long as it's a tiny JS implementation
